cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst

```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
 Note that in the case of the Docker container running inside a Virtual Machine (as on
 either Mac OS X or Microsoft Windows), you will need to substitute the Docker VM's IP
 address for ``localhost`` in the web browser address bar.
+
+
+Customize JVM OPTS
+------------------
+On Linux, you can customize the JVM options for OutOfMemory error and whether to do a heapdump if an OutOfMemory error
```

Comment:
`heapdump` here vs `heap dump` elsewhere.  I *think* it's supposed to be `heap dump`